src/gamebryo/gamegamebryo.cpp

```diff
@@ -425,3 +427,71 @@ QString GameGamebryo::determineMyGamesPath(const QString& gameName)
 
   return {};
 }
+
+QString GameGamebryo::parseEpicGamesLocation(const QStringList& manifests)
+{
+  // Use the registry entry to find the EGL Data dir first, just in case something
+  // changes
+  QString manifestDir = findInRegistry(
+      HKEY_LOCAL_MACHINE, L"SOFTWARE\\WOW6432Node\\Epic Games\\EpicGamesLauncher",
```

Comment:
`WOW6432Node` actually isn't needed. I will remove this later. This is a special directory that's automatically used depending on the bitness context you're querying from the registy with.

Since the registry reader was corrected to check both, there should be no issue.